flink-clients/src/main/java/org/apache/flink/client/CliFrontend.java

```diff
@@ -569,6 +571,69 @@ public int compare(JobStatusMessage o1, JobStatusMessage o2) {
 	}
 
 	/**
+	 * Executes the STOP action.
+	 * 
+	 * @param args Command line arguments for the stop action.
+	 */
+	protected int stop(String[] args) {
+		LOG.info("Running 'stop' command.");
+
+		StopOptions options;
+		try {
+			options = CliFrontendParser.parseStopCommand(args);
+		}
+		catch (CliArgsException e) {
+			return handleArgException(e);
+		}
+		catch (Throwable t) {
+			return handleError(t);
+		}
+
+		// evaluate help flag
+		if (options.isPrintHelp()) {
+			CliFrontendParser.printHelpForStop();
+			return 0;
+		}
+
+		String[] stopArgs = options.getArgs();
+		JobID jobId;
+
+		if (stopArgs.length > 0) {
+			String jobIdString = stopArgs[0];
+			try {
+				jobId = new JobID(StringUtils.hexStringToByte(jobIdString));
+			}
+			catch (Exception e) {
+				LOG.error("Error: The value for the Job ID is not a valid ID.");
```

Comment:
We could. But `cancel` does not use it either. I guess we should have consistent patterns... I would just keep it as is. If you want to get it changed, we should changed it in `cancel`, too.